modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/type/ReferralsOfferInfo.kt

```diff
@@ -0,0 +1,13 @@
+package au.com.shiftyjelly.pocketcasts.models.type
+
+interface ReferralsOfferInfo {
```

Comment:
Matches [iOS class.](https://github.com/Automattic/pocket-casts-ios/pull/2162/files#diff-e853cfa7bdb39b0b46f314d6b730648738fa1f9526b398870eec54e0a203f1f4)

Mock data class used in displays will be replaced with data from Play Store offer.